Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG GO_VERSION
+
+FROM golang:${GO_VERSION}-buster
+
+ARG GO_VERSION
```

Comment:
duplicate

Reply:
no, it's not, but it seems like it.  All `ARG`s that happen prior to the `FROM` statement are lost, so you have to re-state it.  They don't move through stages, so technically, using it before `FROM` is a different stage.

Reply:
I knew I should have saved the place where I read that in the docs.  Can't find it now.

Reply:
I believe you, and TIL!